WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemDataSource.kt

```diff
@@ -94,7 +97,12 @@ class PostListItemDataSource(
     private fun transformToPostListItemType(localOrRemoteId: LocalOrRemoteId, post: PostModel?): PostListItemType =
             if (post == null) {
                 // If the post is not in cache, that means we'll be loading it
-                LoadingItem(localOrRemoteId)
+                val options = if (postListType == TRASHED) {
+                    LoadingItemOptions(false, false, false, true, true)
```

Comment:
NitPick: Lint is complaining here: `Boolean literal argument without parameter name`. I think we might want to consider adding the parameters names to make the code readable even on GitHub where the AS doesn't provide the parameter names.
Another option: I would have probably created two instances (objects) of a sealed class instead of passing 5 boolean parameters from the client's code. But it's just a matter of personal preference. I just wanted to share my approach.

Reply:
Updated in 0137427c7915f6bf429739dd11dd2aa56feb6aff